java/src/jmri/util/swing/JmriBeanComboBox.java

```diff
@@ -35,7 +35,7 @@
     /**
      * Create a default Jmri Combo box for the given bean manager.
      *
-     * @param manager the jmri manager that is used to populate the combo box
+     * @param inManager the jmri manager that is used to populate the combo box
```

Comment:
@geowar1 Had to correct the Javadoc param variable names.
jmri.Manager now imported at the start.